kbase-extension/static/kbase/js/widgets/appWidgets2/input/dynamicDropdownInput.js

```diff
@@ -141,8 +169,20 @@ define([
             });
         }
 
+        function genericClientCall(call_params) {
+            var swUrl = runtime.config('services.workspace.url').replace('ws', 'service_wizard'),
```

Comment:
There's an actual service_wizard url that should be used. It should be in `services.service_wizard.url`

Reply:
Yeah, bad copy paste.